### ISSUE TITLE
librealsense@2.14.0 wrong url

### ISSUE THREAD

**Issue by claudiofantacci** [Homebrew/homebrew-core#30832]
- [x] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, official formula (not cask)? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

#### `brew config`
```
HOMEBREW_VERSION: 1.7.1-127-ged16b04
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: ed16b04c2604060a6da5abac6480c7680f3e21b2
Last commit: 3 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 13c38fc87b0a0efc5da1230b5ea20baad3d69166
Core tap last commit: 27 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_DEV_CMD_RUN: 1
CPU: octa-core 64-bit haswell
Homebrew Ruby: 2.3.7 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/bin/ruby
Clang: 9.0 build 900
Git: 2.17.0 => /usr/local/bin/git
Curl: 7.54.0 => /usr/bin/curl
Java: 1.8.0_25
macOS: 10.13.4-x86_64
CLT: 9.2.0.0.1.1510905681
Xcode: 9.2
XQuartz: 2.7.11 => /opt/X11
```

#### `brew doctor`
```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: A newer Command Line Tools release is available.
Update them from Software Update in the App Store.


Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and what additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew provided
script of the same name. We found the following "config" scripts:
  /Users/Claudio/code-install/bin/yarp-config

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libSOIL.1.07.dylib

Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/SOIL/SOIL.h
  /usr/local/include/arpackpp/acompsol.h
  /usr/local/include/arpackpp/ansymsol.h
  /usr/local/include/arpackpp/arbgcomp.h
  /usr/local/include/arpackpp/arbgnsym.h
  /usr/local/include/arpackpp/arbgsym.h
  /usr/local/include/arpackpp/arbnsmat.h
  /usr/local/include/arpackpp/arbnspen.h
  /usr/local/include/arpackpp/arbscomp.h
  /usr/local/include/arpackpp/arbsmat.h
  /usr/local/include/arpackpp/arbsnsym.h
  /usr/local/include/arpackpp/arbspen.h
  /usr/local/include/arpackpp/arbssym.h
  /usr/local/include/arpackpp/arcgsym.h
  /usr/local/include/arpackpp/arch.h
  /usr/local/include/arpackpp/arcomp.h
  /usr/local/include/arpackpp/arcsmat.h
  /usr/local/include/arpackpp/arcspen.h
  /usr/local/include/arpackpp/arcssym.h
  /usr/local/include/arpackpp/ardfmat.h
  /usr/local/include/arpackpp/ardgcomp.h
  /usr/local/include/arpackpp/ardgnsym.h
  /usr/local/include/arpackpp/ardgsym.h
  /usr/local/include/arpackpp/ardnsmat.h
  /usr/local/include/arpackpp/ardnspen.h
  /usr/local/include/arpackpp/ardscomp.h
  /usr/local/include/arpackpp/ardsmat.h
  /usr/local/include/arpackpp/ardsnsym.h
  /usr/local/include/arpackpp/ardspen.h
  /usr/local/include/arpackpp/ardssym.h
  /usr/local/include/arpackpp/areig.h
  /usr/local/include/arpackpp/arerror.h
  /usr/local/include/arpackpp/argcomp.h
  /usr/local/include/arpackpp/argeig.h
  /usr/local/include/arpackpp/argnsym.h
  /usr/local/include/arpackpp/argsym.h
  /usr/local/include/arpackpp/arhbmat.h
  /usr/local/include/arpackpp/arlcomp.h
  /usr/local/include/arpackpp/arlgcomp.h
  /usr/local/include/arpackpp/arlgnsym.h
  /usr/local/include/arpackpp/arlgsym.h
  /usr/local/include/arpackpp/arlnames.h
  /usr/local/include/arpackpp/arlnsmat.h
  /usr/local/include/arpackpp/arlnspen.h
  /usr/local/include/arpackpp/arlscomp.h
  /usr/local/include/arpackpp/arlsmat.h
  /usr/local/include/arpackpp/arlsnsym.h
  /usr/local/include/arpackpp/arlspdef.h
  /usr/local/include/arpackpp/arlspen.h
  /usr/local/include/arpackpp/arlssym.h
  /usr/local/include/arpackpp/arlsupm.h
  /usr/local/include/arpackpp/arlutil.h
  /usr/local/include/arpackpp/armat.h
  /usr/local/include/arpackpp/arpackf.h
  /usr/local/include/arpackpp/arrgcomp.h
  /usr/local/include/arpackpp/arrgeig.h
  /usr/local/include/arpackpp/arrgnsym.h
  /usr/local/include/arpackpp/arrgsym.h
  /usr/local/include/arpackpp/arrscomp.h
  /usr/local/include/arpackpp/arrseig.h
  /usr/local/include/arpackpp/arrsnsym.h
  /usr/local/include/arpackpp/arrssym.h
  /usr/local/include/arpackpp/arscomp.h
  /usr/local/include/arpackpp/arseig.h
  /usr/local/include/arpackpp/arsnsym.h
  /usr/local/include/arpackpp/arssym.h
  /usr/local/include/arpackpp/arugcomp.h
  /usr/local/include/arpackpp/arugnsym.h
  /usr/local/include/arpackpp/arugsym.h
  /usr/local/include/arpackpp/arunsmat.h
  /usr/local/include/arpackpp/arunspen.h
  /usr/local/include/arpackpp/aruscomp.h
  /usr/local/include/arpackpp/arusmat.h
  /usr/local/include/arpackpp/arusnsym.h
  /usr/local/include/arpackpp/aruspen.h
  /usr/local/include/arpackpp/arussym.h
  /usr/local/include/arpackpp/asymsol.h
  /usr/local/include/arpackpp/bcmatrxa.h
  /usr/local/include/arpackpp/bcmatrxb.h
  /usr/local/include/arpackpp/bcmatrxc.h
  /usr/local/include/arpackpp/blas1c.h
  /usr/local/include/arpackpp/blas1f.h
  /usr/local/include/arpackpp/bnmatrxa.h
  /usr/local/include/arpackpp/bnmatrxb.h
  /usr/local/include/arpackpp/bnmatrxc.h
  /usr/local/include/arpackpp/bnmatrxd.h
  /usr/local/include/arpackpp/bnmatrxe.h
  /usr/local/include/arpackpp/bnmatrxw.h
  /usr/local/include/arpackpp/bsmatrxa.h
  /usr/local/include/arpackpp/bsmatrxb.h
  /usr/local/include/arpackpp/bsmatrxc.h
  /usr/local/include/arpackpp/caupp.h
  /usr/local/include/arpackpp/ceupp.h
  /usr/local/include/arpackpp/cgenprba.h
  /usr/local/include/arpackpp/cgenprbb.h
  /usr/local/include/arpackpp/cholmodc.h
  /usr/local/include/arpackpp/cmatrixa.h
  /usr/local/include/arpackpp/cmatrixb.h
  /usr/local/include/arpackpp/cmatrixc.h
  /usr/local/include/arpackpp/cmatrixd.h
  /usr/local/include/arpackpp/cmatrixe.h
  /usr/local/include/arpackpp/cmatrixf.h
  /usr/local/include/arpackpp/compgsol.h
  /usr/local/include/arpackpp/compsol.h
  /usr/local/include/arpackpp/dcmatrxa.h
  /usr/local/include/arpackpp/dcmatrxb.h
  /usr/local/include/arpackpp/dcmatrxc.h
  /usr/local/include/arpackpp/debug.h
  /usr/local/include/arpackpp/dnmatrxa.h
  /usr/local/include/arpackpp/dnmatrxb.h
  /usr/local/include/arpackpp/dnmatrxc.h
  /usr/local/include/arpackpp/dnmatrxd.h
  /usr/local/include/arpackpp/dnmatrxe.h
  /usr/local/include/arpackpp/dnmatrxw.h
  /usr/local/include/arpackpp/dsmatrxa.h
  /usr/local/include/arpackpp/dsmatrxb.h
  /usr/local/include/arpackpp/dsmatrxc.h
  /usr/local/include/arpackpp/lapackc.h
  /usr/local/include/arpackpp/lapackf.h
  /usr/local/include/arpackpp/lcmatrxa.h
  /usr/local/include/arpackpp/lcmatrxb.h
  /usr/local/include/arpackpp/lcmatrxe.h
  /usr/local/include/arpackpp/lcmatrxf.h
  /usr/local/include/arpackpp/lcompsol.h
  /usr/local/include/arpackpp/lnmatrxa.h
  /usr/local/include/arpackpp/lnmatrxb.h
  /usr/local/include/arpackpp/lnmatrxc.h
  /usr/local/include/arpackpp/lnmatrxd.h
  /usr/local/include/arpackpp/lnmatrxe.h
  /usr/local/include/arpackpp/lnmatrxf.h
  /usr/local/include/arpackpp/lnmatrxv.h
  /usr/local/include/arpackpp/lnmatrxw.h
  /usr/local/include/arpackpp/lnsymsol.h
  /usr/local/include/arpackpp/lsmatrxa.h
  /usr/local/include/arpackpp/lsmatrxb.h
  /usr/local/include/arpackpp/lsmatrxc.h
  /usr/local/include/arpackpp/lsmatrxd.h
  /usr/local/include/arpackpp/lsvdsol.h
  /usr/local/include/arpackpp/lsymsol.h
  /usr/local/include/arpackpp/matprod.h
  /usr/local/include/arpackpp/naupp.h
  /usr/local/include/arpackpp/neupp.h
  /usr/local/include/arpackpp/ngenprba.h
  /usr/local/include/arpackpp/ngenprbb.h
  /usr/local/include/arpackpp/ngenprbc.h
  /usr/local/include/arpackpp/nmatrixa.h
  /usr/local/include/arpackpp/nmatrixb.h
  /usr/local/include/arpackpp/nmatrixc.h
  /usr/local/include/arpackpp/nmatrixd.h
  /usr/local/include/arpackpp/nmatrixe.h
  /usr/local/include/arpackpp/nmatrixf.h
  /usr/local/include/arpackpp/nmatrixv.h
  /usr/local/include/arpackpp/nsymgsol.h
  /usr/local/include/arpackpp/nsymsol.h
  /usr/local/include/arpackpp/nsymvsol.h
  /usr/local/include/arpackpp/rcompgsl.h
  /usr/local/include/arpackpp/rcompsol.h
  /usr/local/include/arpackpp/rnsymgsl.h
  /usr/local/include/arpackpp/rnsymsol.h
  /usr/local/include/arpackpp/rnsymvsl.h
  /usr/local/include/arpackpp/rsymgsol.h
  /usr/local/include/arpackpp/rsymsol.h
  /usr/local/include/arpackpp/saupp.h
  /usr/local/include/arpackpp/seupp.h
  /usr/local/include/arpackpp/sgenprba.h
  /usr/local/include/arpackpp/sgenprbb.h
  /usr/local/include/arpackpp/smatrixa.h
  /usr/local/include/arpackpp/smatrixb.h
  /usr/local/include/arpackpp/smatrixc.h
  /usr/local/include/arpackpp/smatrixd.h
  /usr/local/include/arpackpp/superluc.h
  /usr/local/include/arpackpp/symgsol.h
  /usr/local/include/arpackpp/symsol.h
  /usr/local/include/arpackpp/umfpackc.h

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libSOIL.a

Warning: You have unlinked kegs in your Cellar
Leaving kegs unlinked can lead to build-trouble and cause brews that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  ogre
  opencv
  gnupg

Warning: Your Xcode (9.2) is outdated.
Please update to Xcode 9.4 (or delete it).
Xcode can be updated from the App Store.
```
---

Issuing a `brew install librealsense --with-example` you get the following error:
```
==> Downloading https://github.com/IntelRealSense/librealsense/archive/2.14.0.tar.gz
==> Downloading from https://codeload.github.com/IntelRealSense/librealsense/tar.gz/2.14.0

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "librealsense"
Download failed: https://github.com/IntelRealSense/librealsense/archive/2.14.0.tar.gz
```

The url should be `https://github.com/IntelRealSense/librealsense/archive/v2.14.0.tar.gz`. There is a missing `v` before the version number.

I can both fix and update the formula to the current latest release `2.14.1`.

**Comment by SMillerDev** [comment]
> I can both fix and update the formula to the current latest release 2.14.1. 

Making a pull request would be best, thanks

**Comment by claudiofantacci** [comment]
> Making a pull request would be best, thanks

PR submitted 👍

**Comment by claudiofantacci** [comment]
Closed via commit b7a29cb and 7abca79.
